cloud/linode/cloud.go

```diff
@@ -98,7 +100,15 @@ func newCloud() (cloudprovider.Interface, error) {
 	url := os.Getenv(urlEnv)
 	ua := fmt.Sprintf("linode-cloud-controller-manager %s", linodego.DefaultUserAgent)
 
-	linodeClient, err := client.New(apiToken, ua, url)
+	// set timeout used by linodeclient for API calls
+	timeout := client.DefaultClientTimeout
+	if raw, ok := os.LookupEnv("LINODE_REQUEST_TIMEOUT_SECONDS"); ok {
+		if t, _ := strconv.Atoi(raw); t > 0 {
+			timeout = time.Duration(t) * time.Second
+		}
```

Comment:
nit: This reads a little strangely to me.

Instead, I would recommend keying off of the returned error to know whether or not `t` is usable:

```suggestion
		if t, err := strconv.Atoi(raw); err == nil {
			timeout = time.Duration(t) * time.Second
		}
```

NOTE: Could the `LINODE_REQUEST_TIMEOUT_SECONDS` environment variable be renamed to `LINODE_REQUEST_TIMEOUT` and hold a duration string that is parsed with [time.ParseDuration](https://pkg.go.dev/time#ParseDuration)?

Reply:
Updated the code. We could rename the `LINODE_REQUEST_TIMEOUT_SECONDS` as suggested, just trying to keep the code same as we have at other places, hence keeping it as it is here. Let me know if you still feel we should use time.ParseDuration and I can update this codepath as well.

Reply:
> just trying to keep the code same as we have at other places

That's fair! It was just an idea. :smile:

> Let me know if you still feel we should use time.ParseDuration and I can update this codepath as well.

No, I don't think this is necessary.